### ISSUE TITLE
what is the order of Imagenet categories in the pretrained inception V3? 

### ISSUE THREAD

**Issue by fernande2000** [keras-team/keras#10083]
I am training, just to check my code, Imagenet on the pretrained Imagenet inception V3. However the accuracy I obtain is zero. I suspect this might be due to the fact that the hot vector I input does not match the order in the categories in which the model was trained. I have seen that there are more than one set of category orders out there , see for instance: https://gist.github.com/aaronpolhamus/964a4411c0906315deb9f4a3723aac57
and 
https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a
which one should I use to match the pretrained inception_v3?

**Comment by fernande2000** [comment]
I found it at https://s3.amazonaws.com/deep-learning-models/image-models/imagenet_class_index.json and it does not match my list. I wonder what there are more that one list of label orders!
